graphql/schema.graphql

```diff
@@ -316,6 +317,12 @@ input ProjectInput {
   repo: String!
 }
 
+input CopyProjectInput {
+  projectIdToCopy: String
+  newProjectIdentifier: String!
```

Comment:
Right, we do need to support this which is what I meant by the original description; it was introduced after the designs were created which is why you don't see it there. I don't think we necessarily need to ask Lara to update the design since it's a small thing but we should maybe drop a comment so we remember it should be there?